### ISSUE TITLE
Add wait flags to svcat commands

### ISSUE THREAD

**Issue by carolynvs** [kubernetes-incubator/service-catalog#1736]
A pretty common scenario is that I want to execute an operation (provision/bind) and then wait for it to be ready to use. I think it would be very dev friendly to support those with flags, instead of driving people to use bash-foo:

* `svcat OP` does not wait by default. The status will be empty in the output. Useful when you want to chain multiple commands and wait for them later.
* `svcat OP --wait [--timeout=5m]` waits for the instance to be provisioned or fail. Useful for simple scripts that aren't performing concurrent tasks. When a timeout is not set, wait forever.
* `svcat get RESOURCE --watch` mimics the functionality of `kubectl`. Useful for those who didn't use `--wait` on the original OP or are watching an operation they didn't initiate from svcat.
* Use a slight backoff instead of polling at a set interval. We shouldn't be so aggressive on the backoff that we aren't checking status for huge spans of time. Just enough that we aren't polling every second the entire time.

Examples:

```
svcat provision mysql --wait

svcat bind mysql
svcat get binding mysql --watch
```

Migrated from https://github.com/Azure/service-catalog-cli/issues/89

**Comment by duglin** [comment]
I would also think it should be the opposite - make the default "wait" and add some "--no-wait" flag for the less common cases where you don't want to wait.

**Comment by carolynvs** [comment]
I agree! Not sure why I picked it to work that way when I first created that issue a while back but I'd prefer that svcat do what most people probably want by default.

**Comment by carolynvs** [comment]
#dibs

**Comment by mrbobbytables** [comment]
This project is being archived, closing open issues and PRs. 
Please see this PR for more information: https://github.com/kubernetes/community/pull/6632
